R/run.R

```diff
@@ -283,6 +295,74 @@ CmdStanRun <- R6::R6Class(
 }
 CmdStanRun$set("private", name = "run_sample_", value = .run_sample)
 
+.run_generate_quantities <- function() {
+  procs <- self$procs
+  on.exit(procs$cleanup(), add = TRUE)
+
+  # add path to the TBB library to the PATH variable
+  if (cmdstan_version() >= "2.21" && os_is_windows()) {
+    path_to_TBB <- file.path(cmdstan_path(), "stan", "lib", "stan_math", "lib", "tbb")
+    current_path <- Sys.getenv("PATH")
+    if (regexpr("path_to_TBB", current_path, perl = TRUE) <= 0) {
+      Sys.setenv(PATH = paste0(path_to_TBB, ";", Sys.getenv("PATH")))
+    }
+  }
+  if (procs$num_procs() == 1) {
+    start_msg <- "Running standalone generated quantities with 1 chain"
```

Comment:
Should this be:
`"Running standalone generated quantities for 1 MCMC chain"`
and the same then for others.

Reply:
Changed to:
```r
  if (procs$num_procs() == 1) {
    start_msg <- "Running standalone generated quantities after 1 MCMC chain"
  } else if (procs$num_procs() == procs$parallel_procs()) {
    start_msg <- paste0("Running standalone generated quantities after ", procs$num_procs(), " MCMC chains, for all chains in parallel ")
  } else {
    if (procs$parallel_procs() == 1) {
      start_msg <- paste0("Running standalone generated quantities after ", procs$num_procs(), " MCMC chains, for 1 chain at a time ")
    } else {
      start_msg <- paste0("Running standalone generated quantities after ", procs$num_procs(), " MCMC chains, for ", procs$parallel_procs(), " chains at a time ")
    }
  }
```
but I am not sure its the best idea. Happy to hear any suggestions.

Reply:
How about what you have but without "for". That sounds a bit better to me:

```
"Running standalone generated quantities after 4 MCMC chains, all chains in parallel "
"Running standalone generated quantities after 4 MCMC chains, 1 chain at a time "
"Running standalone generated quantities after 4 MCMC chains, 2 chain at a time "
```

Reply:
Done. Thanks!